src/components/Badge/tests/Badge.test.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import {mount} from 'enzyme';
+import Badge from '../Badge';
+
+describe('<DisplayText />', () => {
+
+    it('Verify the default badge.', () => {
```

Comment:
really minor suggestion for readability.
it('verifies the default badge.', () => {